src/lib.rs

```diff
@@ -169,11 +169,11 @@ const QUORUM_NUMERATOR: usize = 2;
 /// See `QUORUM_NUMERATOR`.
 const QUORUM_DENOMINATOR: usize = 3;
 
-/// Minimal safe section size. Routing will keep adding nodes until the section reaches this size.
+/// Recommented section size. Routing will keep adding nodes until the section reaches this size.
 /// More nodes might be added if requested by the upper layers.
 /// This number also detemines when split happens - if both post-split sections would have at least
 /// this number of nodes.
-const SAFE_SECTION_SIZE: usize = 100;
+const RECOMMENDED_SECTION_SIZE: usize = 100;
```

Comment:
OK

Reply:
Wonder if we should maybe at some point try to run some high-level network simulations to determine optimal value for this.

Reply:
It's a tough one as we can simulate the network storage side, but we would have to guess at mutations and number of clients and so on. The number of clients is the one I am wary of if the section is large. I would prefer lots of small sections, just to allow Elders to cope with mutations and clients. 
We need to work out something to check/test this though and with sections only adding nodes when vaults tell routing then it should mean the first section is close in work to the nth section, at least I imagine it will be.

Reply:
Yeah, I see. Maybe we can experiment with it on alpha.